src/test/java/htmlverification/tests/CvsHgvPRSBilingualTest.java

```diff
@@ -0,0 +1,220 @@
+import static uk.gov.dvsa.model.cvs.certificateData.CvsMotFailCertificateData.FAILED_SUMMARY_HEADER;
+import static uk.gov.dvsa.model.cvs.certificateData.CvsMotFailCertificateDataWelsh.FAILED_SUMMARY_HEADER_WELSH;
+import static uk.gov.dvsa.model.mot.results.Summary.EU_NUMBER_SUMMARY_HEADER;
+
+public class CvsHgvPRSBilingualTest {
+
+    protected HtmlGenerator htmlGenerator;
+    protected CvsHgvPRSBilingual testCertificate;
+    protected CertificatePageObject certificatePageObjectVTG5;
+    protected CertificatePageObject certificatePageObjectVTG30;
+
+    protected CertificatePageObject certificatePageObjectVTG5W;
+    protected CertificatePageObject certificatePageObjectVTG30W;
+
+    public CvsHgvPRSBilingualTest() {
+        this.htmlGenerator = new HtmlGenerator(new Handlebars());
+    }
+
+    @Before
+    public void setup() throws IOException {
+        testCertificate = CvsCertificateTestDataProvider.getCvsHgvPRSBilingual();
+
+        String certHtmlVTG5 = htmlGenerator.generate(testCertificate).get(0);
+        String certHtmlVTG30 = htmlGenerator.generate(testCertificate).get(1);
+        String certHtmlVTV5W = htmlGenerator.generate(testCertificate).get(2);
+        String certHtmlVTG30W = htmlGenerator.generate(testCertificate).get(3);
+
+        certificatePageObjectVTG5 = new CertificatePageObject(certHtmlVTG5);
+        certificatePageObjectVTG30 = new CertificatePageObject(certHtmlVTG30);
+        certificatePageObjectVTG5W = new CertificatePageObject(certHtmlVTV5W);
+        certificatePageObjectVTG30W = new CertificatePageObject(certHtmlVTG30W);
+    }
+
+    @Test
+    public void verifyResultSummaries() {
+        String resultName = certificatePageObjectVTG5.getDefectSummaryComponent().getResultNameItem().text();
+        String resultNameFail = certificatePageObjectVTG30.getDefectSummaryComponent().getResultNameItem().text();
+        String resultNameWelsh = certificatePageObjectVTG5W.getDefectSummaryComponent().getResultNameItem().text();
+        String resultNameFailWelsh = certificatePageObjectVTG30W.getDefectSummaryComponent().getResultNameItem().text();
+
+        assertEquals(
+                String.format("%s%s%s %s", "(", EU_NUMBER_SUMMARY_HEADER, ")", PASS_SUMMARY_HEADER),
+                resultName
+        );
+        assertEquals(
+                String.format("%s%s%s %s", "(", EU_NUMBER_SUMMARY_HEADER, ")", FAILED_SUMMARY_HEADER),
+                resultNameFail
+        );
+        assertEquals(
+                String.format("%s%s%s %s", "(", EU_NUMBER_SUMMARY_HEADER, ")", PASS_SUMMARY_HEADER_WELSH),
+                resultNameWelsh
+        );
+        assertEquals(
+                String.format("%s%s%s %s", "(", EU_NUMBER_SUMMARY_HEADER, ")", FAILED_SUMMARY_HEADER_WELSH),
+                resultNameFailWelsh
+        );
+    }
+
+    @Test
+    public void verifyVin() {
+        String vin = certificatePageObjectVTG5.getVin();
```

Comment:
It's probably better for these to be named vinPass to make it clear vs vinFail

Reply:
updated